### ISSUE TITLE
Add codecov token for tests

### ISSUE THREAD

**Issue by mpadge** [hypertidy/geodist#41]
@mdsumner Assigning this one to you, because you've got the keys to https://app.codecov.io/gh/hypertidy. Can you please get a token from there and put in this repo so that coverage reports can be uploaded again? Thanks!

**Comment by mdsumner** [comment]
Sorry I totally missed this. Have I done it?  I created `CODECOV_TOKEN` on https://app.codecov.io/ but not sure if I have to set it here too? Just add an env-var?

**Comment by mpadge** [comment]
Yeah, you need to set an envvar, which you can also do for the whole org. The codecov docs suggest org-wide as the best way, but i've had a few flaky cases where that doesn't get picked up, so usually end up setting for each repo anyway. Thanks!

**Comment by mdsumner** [comment]
I don't find it at all intuitive, whether I need to create an environment or just add a repo secret. 

I added it as a Repository Secret
